src/core/importType.js

```diff
@@ -49,6 +49,10 @@ function isRelativeToSibling(name) {
   return name.indexOf('./') === 0
 }
 
+export function isPrivateModule(name) {
+  return name.indexOf('@') === 0
```

Comment:
also, the name isn't "private", it's "scoped" - scoped modules can be either public or private.